sensu-plugins-chef.gemspec

```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |s|
     s.add_runtime_dependency 'rack', '~> 1.6.5'
   end
 
-  s.add_runtime_dependency 'ridley',       '4.1.2'
+  s.add_runtime_dependency 'ridley',       '5.1.0'
   s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
   s.add_runtime_dependency 'varia_model', '0.4.1'
```

Comment:
looks like this needs to be bumped per travis: https://travis-ci.org/sensu-plugins/sensu-plugins-chef/jobs/230068843